common/src/Model/Node.h

```diff
@@ -56,7 +56,20 @@ namespace TrenchBroom {
             virtual ~Node();
         public: // getters
             const String& name() const;
+            /**
+             * Returns a box that encloses the node and its children. These are the bounds that are relevant
+             * for gameplay (for entities, the bounds specified in the entity definition file), and used
+             * for grid snapping, for example.
+             * Nodes can render or hit test outside of these bounds if necessary (see `cullingBounds()`).
+             */
```

Comment:
I agree that it would be better to rename this to something more specific that differentiates this concept better from the other bounds (now called `cullingBounds`). But here I would say that these are the "logical" bounds, whereas the other bounds are something like the "actual" or "physical" bounds?

Reply:
logicalBounds / physicalBounds aren't my top pick because these are overloaded a lot in computing, e.g. in networking or high DPI graphics. 

The inner bounds could be called: `gameplayBounds` (it's what the game uses), `alignmentBounds` (it's what is used for aligning things)
The outer bounds: `visualBounds`, `hitTestingBounds`, `pickingBounds` ,`totalBounds` or just `bounds`? I dunno..

Reply:
I prefer the names to reflect what the bounds represent, not what they are used for. Hence my suggestion of logical and physical bounds. I'll think about it a bit more, maybe I'll be able to come up with something else.

Reply:
Ok, let's go with logicalBounds/physicalBounds for now. If you think of something better it's a quick refactor.

Reply:
Alright!